generators/app/modules/prompts.js

```diff
@@ -5,6 +5,16 @@ function notBlank(input) {
   return "Please enter a description.";
 }
 
+// https://gist.github.com/mathewbyrne/1280286
+function slugify(text) {
```

Comment:
Can we move this to an external file that we require to help keep this `prompts` file as straightforward as possible?